### PR TITLE
Update mTLS guide

### PR DESCRIPTION
**Pull request by alexfornuto**
## Summary

Update with copy edits and formatted. Tech required no changes.

## Related issues

Resolves https://github.com/pomerium/docs/issues/52
Resolves https://github.com/pomerium/docs/issues/10

## Checklist

- [x] reference any related issues
- [x] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review

### CODE REVIEW

**Comment by alexfornuto** [comment]
Hold of on review, adding more content.

**Comment by alexfornuto** [comment]
Ready for review.

File: docs/partials/client-cert-alt-port.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Route doesn't ask for certificate
```
Comment (travisgroth) [inline docs/partials/client-cert-alt-port.md:1]: Since this is fixed in the next release, maybe remove this warning and don't backport the guide?

**Review by travisgroth — CHANGES_REQUESTED** [review]
Left suggestions/comments

File: docs/guides/mtls.md [hunk 2]
```diff
@@ -15,103 +15,143 @@ Secure communication on the web typically refers to using signed server certific
...
 ```yaml
-# config.yaml
-address: ":8443"
-authenticate_service_url: "https://authenticate.localhost.pomerium.io:8443"
+# If you're using a separate certificate for server-side TLS, leave these keys unchanged.
 certificate_file: "_wildcard.localhost.pomerium.io.pem"
 certificate_key_file: "_wildcard.localhost.pomerium.io-key.pem"
 
 # "$(mkcert -CAROOT)/rootCA.pem"
 client_ca_file: "/YOUR/MKCERT/CAROOT/rootCA.pem"
```
[the remark this answers is not in the document]
Comment (alexfornuto) [reply docs/guides/mtls.md:88]: Per discussion, resolved by ecb0eb69d162d774156465a14e1b1a5bc279ec82

File: docs/partials/client-cert-alt-port.md (hunk 1 above)
Comment (alexfornuto) [reply docs/partials/client-cert-alt-port.md:1]: resolved in 0f72ec24997ff2a857897f642002ed356d336223, though we will also need to update the Troubleshooting page for the same thing.

**Review by travisgroth — APPROVED** [review]

**Review by desimone — APPROVED** [review]
Left commentary https://github.com/pomerium/pomerium/pull/2814#pullrequestreview-838924548; approved otherwise

**Pull request merged**